jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_ru.properties

```diff
@@ -62,6 +62,7 @@ label.answer.italic=\u041A\u0443\u0440\u0441\u0438\u0432 (Ctrl+I)
 label.answer.underline=\u041F\u043E\u0434\u0447\u0435\u0440\u043A\u043D\u0443\u0442\u044B\u0439 (Ctrl+U)
 label.answer.striked=\u0417\u0430\u0447\u0435\u0440\u043A\u043D\u0443\u0442\u044B\u0439 (Ctrl+S)
 label.answer.highlight=\u041F\u043E\u0434\u0441\u0432\u0435\u0442\u043A\u0430
+label.answer.highlight.button=\u041f\u043e\u0434\u0441\u0432\u0435\u0442\u043a\u0430 \u0442\u0435\u043a\u0441\u0442\u0430
```

Comment:
@RuslanSh , could you please also correct the tooltips for the highlight button (in all languages): `label.answer.highlight`